### PR TITLE
Update Dutch translation

### PR DESCRIPTION
**Pull request by coreice**
Added over 200 new strings and removed obsolete strings from older (pre 1.3) Bracket versions. Edited some existing strings to be more accurate. Tried to be as consistent as possible when translating certain jargon.

### CODE REVIEW

**Comment by MarcelGerber** [comment]
Thanks for your contribution!
You need to sign the [contributor license agreement](http://dev.brackets.io/brackets-contributor-license-agreement.html) first, we can't take your changes otherwise.

**Comment by abose** [comment]
@coreice As brackets is an open source project, The  contributor license agreement is for your protection as well as the protection of Brackets and its licensees; it does not change your rights to use your own Contributions for any other purpose.

**Comment by coreice** [comment]
@abose I'm aware of that. I hope my contribution will be approved.

**Comment by abose** [comment]
@coreice have you signed the CLA?

**Comment by MarcelGerber** [comment]
@abose He has (according to `grunt cla-check --user=coreice`), but the Travis build fails either way for some reason. I've restarted it twice now without success.

File: src/nls/nl/strings.js [hunk 1]
```diff
@@ -103,63 +129,126 @@ define({
...
+    "LEFT"              : "Links",
+    "RIGHT"             : "Rechts", 
+    
+    "CMD_SPLITVIEW_NONE"        : "Niet splitsen",
+    "CMD_SPLITVIEW_VERTICAL"    : "Verticaal splitsen",
+    "CMD_SPLITVIEW_HORIZONTAL"  : "Horizontaal splitsen",
+    "SPLITVIEW_MENU_TOOLTIP"    : "Splits de editor verticaal of horizontaal",
+    "GEAR_MENU_TOOLTIP"         : "Configureer bestanden",
+    
+     "SPLITVIEW_INFO_TITLE"              : "Al geopend",
```
Comment (MarcelGerber) [inline src/nls/nl/strings.js]: nit: this line is indented one space too much

File: src/nls/nl/strings.js [hunk 2]
```diff
@@ -311,41 +434,56 @@ define({
...
     "ABOUT_TEXT_LINE4"                     : "Documentatie en broncode op <a href='https://github.com/adobe/brackets/'>https://github.com/adobe/brackets/</a>",
     "ABOUT_TEXT_LINE5"                     : "Gemaakt met \u2764 en JavaScript door:",
     "ABOUT_TEXT_LINE6"                     : "Veel mensen (maar we hebben problemen met het laden van die data op dit moment).",
-    "ABOUT_TEXT_WEB_PLATFORM_DOCS"         : "Web Platform Docs en het Web Platform logo zijn gelicentieerd onder een Creative Commons Attribution licentie, <a href='{WEB_PLATFORM_DOCS_LICENSE}'>CC-BY 3.0 Unported</a>.",
-    "UPDATE_NOTIFICATION_TOOLTIP"          : "Er is een nieuwe versie van {APP_NAME} beschikbaar! Klik hier voor details.",
+      "ABOUT_TEXT_WEB_PLATFORM_DOCS"         : "Web Platform Docs en het Web Platform logo zijn gelicentieerd onder een Creative Commons Attribution licentie, <a href='{WEB_PLATFORM_DOCS_LICENSE}'>CC-BY 3.0 Unported</a>.",
+      "UPDATE_NOTIFICATION_TOOLTIP"          : "Er is een nieuwe versie van {APP_NAME} beschikbaar! Klik hier voor details.",
```
Comment (MarcelGerber) [inline src/nls/nl/strings.js]: nit: remove the extra leading whitespace for these two lines

**Comment by MarcelGerber** [comment]
@ispieter @githrdw Would you mind doing a quick review on these changes & additions to the Dutch translation?

**Comment by abose** [comment]
@coreice Could you update this PR.

**Comment by ispieter** [comment]
Yes, I see several errors in the translations. @abose, what is the best way to mark these? Should I leave a comment on all of them?

**Comment by coreice** [comment]
If you can mark these errors I will fix them.

**Comment by MarcelGerber** [comment]
@ispieter It's best to add a line note to every line you'd like to be changed.
It'd be very helpful if you could include both an explanation of what's wrong in your opinion and a suggested correction.

@coreice Please make sure you fix the linting errors and the missppelled key, too.

File: src/nls/nl/strings.js [hunk 3]
```diff
@@ -103,63 +129,126 @@ define({
...
+    "ERROR_FETCHING_UPDATE_INFO_MSG"    : "Er is een fout opgetreden bij het ophalen van de laatste update-informatie van de server. Zorg ervoor dat je verbonden bent met het internet en probeer opnieuw.",    
+    
+    // File exclusion filters
+    "NEW_FILE_FILTER"                   : "Nieuwe set uitgesloten bestanden\u2026",
+    "CLEAR_FILE_FILTER"                 : "Sluit geen bestanden uit",
+    "NO_FILE_FILTER"                    : "Geen uitgesloten bestanden",
+    "EXCLUDE_FILE_FILTER"               : "Sluit {0} uit",
+    "EDIT_FILE_FILTER"                  : "Bewerk\u2026",
+    "FILE_FILTER_DIALOG"                : "Bewerk set uitgesloten bestanden",
+    "FILE_FILTER_INSTRUCTIONS"          : "Sluit bestanden en mappen uit met de volgende stringen/substringen of <a href='{0}' title='{0}'>wildcards</a>. Plaats elke string op een nieuwe regel.",
```
Comment (ispieter) [inline src/nls/nl/strings.js]: 'stringen/substringen' are no Dutch words. Perhaps 'teksten/deelteksten' is a better alternative.

File: src/nls/nl/strings.js [hunk 4]
```diff
@@ -251,6 +355,20 @@ define({
...
 
+    // Search menu commands
+    "FIND_MENU"                           : "Zoek", 
+    "CMD_FIND"                            : "Zoek",
+    "CMD_FIND_NEXT"                       : "Zoek volgende",
+    "CMD_FIND_PREVIOUS"                   : "Zoek vorige",
+    "CMD_FIND_ALL_AND_SELECT"             : "Zoek alles en selecteer",
+    "CMD_ADD_NEXT_MATCH"                  : "Voeg eerstvolgende overeenkomst toe aan selectie",
+    "CMD_SKIP_CURRENT_MATCH"              : "Sla-over en selecteer volgende overeenkomst",   
+    "CMD_FIND_IN_FILES"                   : "Zoek in mappen",
```
Comment (ispieter) [inline src/nls/nl/strings.js]: Should be 'Zoek in bestanden' as we're referring to files.

File: src/nls/nl/strings.js [hunk 5]
```diff
@@ -265,45 +383,50 @@ define({
...
     "CMD_QUICK_OPEN"                      : "Snel openen",
-    "CMD_GOTO_LINE"                       : "Ga naar regel",
+    "CMD_GOTO_LINE"                       : "Ga naar regel\u2026",
     "CMD_GOTO_DEFINITION"                 : "Definitie snel zoeken",
     "CMD_GOTO_FIRST_PROBLEM"              : "Ga naar de eerstvolgende fout/waarschuwing",
     "CMD_TOGGLE_QUICK_EDIT"               : "Snel wijzigen",
     "CMD_TOGGLE_QUICK_DOCS"               : "Snel naar documentatie",
     "CMD_QUICK_EDIT_PREV_MATCH"           : "Vorige overeenkomst",
     "CMD_QUICK_EDIT_NEXT_MATCH"           : "Volgende overeenkomst",
+    "CMD_CSS_QUICK_EDIT_NEW_RULE"         : "Nieuwe regels",
```
Comment (ispieter) [inline src/nls/nl/strings.js]: Should be 'Nieuwe regel' as 'Rule' is singular.

File: src/nls/nl/strings.js [hunk 6]
```diff
@@ -265,45 +383,50 @@ define({
...
     "CMD_NEXT_DOC"                        : "Volgend document",
     "CMD_PREV_DOC"                        : "Vorig document",
     "CMD_SHOW_IN_TREE"                    : "Toon in bestandsboom",
+    "CMD_SHOW_IN_EXPLORER"                : "Toon in Explorer",
```
Comment (ispieter) [inline src/nls/nl/strings.js]: 'Explorer' could be renamed to 'Verkenner'. Dutch translation of the Windows file manager.

File: src/nls/nl/strings.js [hunk 7]
```diff
@@ -311,41 +434,56 @@ define({
...
-    "INSTALL_EXTENSION_TITLE"              : "Installeer uitbreiding",
-    "UPDATE_EXTENSION_TITLE"               : "Update uitbreiding",
-    "INSTALL_EXTENSION_LABEL"              : "Uitbreiding URL",
-    "INSTALL_EXTENSION_HINT"               : "URL van het zip bestand of de GitHup repo van de uitbreiding",
-    "INSTALLING_FROM"                      : "Bezig met installeren van uitbreiding van {0}\u2026",
+    "CANT_UPDATE_DEV"                      : "Extensies in de \"dev\" map kunnen niet automatisch geupdate worden.",
+    "INSTALL_EXTENSION_TITLE"              : "Installeer extensie",
+    "UPDATE_EXTENSION_TITLE"               : "Update extensie",
+    "INSTALL_EXTENSION_LABEL"              : "Extensie URL",
+    "INSTALL_EXTENSION_HINT"               : "URL van het zip bestand of de GitHup repo van de extensie",
```
Comment (ispieter) [inline src/nls/nl/strings.js]: 'Githup' should be 'Github'.

File: src/nls/nl/strings.js [hunk 8]
```diff
@@ -355,82 +493,128 @@ define({
...
-    "UPDATE_AND_QUIT"                      : "Update Uitbreidingen en afsluiten",
-    "EXTENSION_NOT_INSTALLED"              : "Kon de uitbreiding {0} niet verwijderen omdat deze niet is geïnstalleerd",
-    "NO_EXTENSIONS"                        : "Geen uitbreidingen geïnstalleerd.<br>Klik op de Beschikbaar tab hierboven om te starten.",
-    "NO_EXTENSION_MATCHES"                 : "Geen uitbreidingen komen overeen met je zoekopdracht.",
-    "REGISTRY_SANITY_CHECK_WARNING"        : "Wees voorzichtig bij het installeren van uitbreidingen van een onbekende bron.",
+    "CHANGE_AND_RELOAD_TITLE"                : "Wijzig extensies",
+    "CHANGE_AND_RELOAD_MESSAGE"              : "Om de aangeduidde extensie te updaten of te verwijderen, moet je {APP_NAME} stoppen en herstarten. Er zal gevraagd worden alle onbewaarde wijzigingen op te slaan.",
+    "REMOVE_AND_RELOAD"                      : "Verwijder extensies en afsluiten",
+    "CHANGE_AND_RELOAD"                      : "Wijzig extensies en afsluiten",
+    "UPDATE_AND_RELOAD"                      : "Update extensies en afsluiten",
```
Comment (ispieter) [inline src/nls/nl/strings.js]: Line 545, 546 and 547 should use 'herladen' instead of the word 'afsluiten'.

File: src/nls/nl/strings.js [hunk 9]
```diff
@@ -355,82 +493,128 @@ define({
...
+    "PROCESSING_EXTENSIONS"                : "Processing extension changes\u2026",
```
Comment (ispieter) [inline src/nls/nl/strings.js]: This is not translated yet.

**Comment by ispieter** [comment]
These are the things that I found. The rest seems fine to me. :)

**Comment by coreice** [comment]
Thanks a lot, I will process them :). I hope it will be ready this week but I've a busy schedule so it also could be next week.

**Comment by abose** [comment]
Thanks @ispieter for reviewing this. @coreice Did you get a chance to look into this? Brackets will have a release within the next two weeks.

**Comment by coreice** [comment]
@abose Just returned home from a convention. Will be working on it now :). Do you want me to create a new edit and pull request after I'm done?

**Comment by abose** [comment]
@coreice :+1: Editing this PR would be good for other reviewers to keep track of the changes.

**Comment by coreice** [comment]
Updated the PR. Thanks for checking @MarcelGerber & @ispieter, there were some stupid mistakes in the previous one. I hope this one will be better :).

**Comment by MarcelGerber** [comment]
@ispieter Does this look good to you?

**Comment by MarcelGerber** [comment]
You should also update the SHA comment at the very bottom. This is based on 50cbe1b8848786c6be27d4788e4b6a367244abc2, I guess.

**Comment by coreice** [comment]
@MarcelGerber You're right. Fixed the SHA comment.

**Comment by ispieter** [comment]
Yes, looks good to me.

**Comment by MarcelGerber** [comment]
Thank you @coreice @ispieter.
Merging.

**Pull request merged**